prog/weaveutil/addrs.go

```diff
@@ -27,10 +27,25 @@ func containerAddrs(args []string) error {
 		return err
 	}
 
+	containerArgs := []string{}
+	for _, cid := range args[1:] {
+		if cid == "weave:allids" { // expand to all container IDs
```

Comment:
`allids` is used in two places: one requires `expose` and the other does not, so I don't think having it do `expose` implicitly is a win.  Maybe you meant change to two flags?